wherehows-web/app/components/datasets/containers/dataset-compliance.ts

```diff
@@ -317,6 +333,17 @@ export default class DatasetComplianceContainer extends Component {
     }
   }
 
+  /**
+   * Persists the updates to the export policy on the remote host
+   * @return {Promise<void}
+   */
+  @action
+  async saveExportPolicy(this: DatasetComplianceContainer, exportPolicy: IDatasetExportPolicy): Promise<void> {
+    await this.notifyOnSave<void>(saveDatasetExportPolicyByUrn(get(this, 'urn'), exportPolicy));
+
+    this.resetPrivacyCompliancePolicy.call(this);
```

Comment:
Why does this need to be done after saving the exportPolicy, or this isn't isolated yet? If not ps add a comment